deps.edn

```diff
@@ -396,8 +396,7 @@
   ;; clojure -T:build uberjar
   ;; clojure -T:build uberjar :edition :ee
   :build
-  {:deps       {io.github.clojure/tools.build   {:git/tag "v0.1.6", :git/sha "5636e61"}
-                com.github.seancorfield/depstar {:mvn/version "2.1.278"}
+  {:deps       {io.github.clojure/tools.build   {:git/tag "v0.5.0", :git/sha "7d77952"}
```

Comment:
this in a few other files too, should we update them all to be on a consistent version?

Reply:
what is in a few other places? tools.build? If so yes that should for sure be updated. i'll check

Reply:
Yes, tools.build

```
bin/build-drivers/deps.edn
10:  io.github.clojure/tools.build   {:git/tag "v0.1.6", :git/sha "5636e61"}

java/deps.edn
10:  {:deps       {io.github.clojure/tools.build {:git/tag "v0.1.6" :git/sha "5636e61"}}

modules/drivers/sparksql/deps.edn
38:  {:deps       {io.github.clojure/tools.build {:git/tag "v0.1.6" :git/sha "5636e61"}}
```